### PR TITLE
Performance improvement

### PR DESCRIPTION
**Pull request by Javivy**
In this milestone we:

-[x] Improved the given queries
-[x] Made use of Indexes

# Results of queries:

# SELECT COUNT(*) FROM visits where animal_id = 4;

Before index:

image

After index: 

image

# SELECT * FROM visits where vet_id = 2;

Before index:

image

After index:

image

# SELECT * FROM owners where email = 'owner_18327@mail.com';

Before index:

image

After index:

image

### CODE REVIEW

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @Javivy  👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

You have missing headers in your README, Kindly make sure that all main headers from the [README template](https://github.com/microverseinc/readme-template) are present  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


Kindly create  a new PR that follows the Gitflow approach

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -36,22 +98,67 @@
...
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
 
-- **SQL Database**
+> Add a link to your deployed project.
+
+- [Live Demo Link](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (omarsalem7) [inline README.md:114]: - [ ] Kindly also remove the live demo section because it is not needed for this project

File: README.md [hunk 2]
```diff
@@ -36,22 +98,67 @@
...
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (omarsalem7) [inline README.md:120]: - [ ] As stated above, kindly remove the placeholder from all sections to make your readme clean and professional

File: README.md [hunk 3]
```diff
@@ -60,24 +167,32 @@
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (omarsalem7) [inline README.md:196]: - [ ] Consider fixing the broken MIT license link and adding your own license and removing the placeholder text under it

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @Javivy and @dgcuenca,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**